packages/sequencer-interactions/test/read-proxy-event.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { expect } from 'chai'
+import { JsonRpcProvider } from '@ethersproject/providers'
+import { Contract, ContractFactory, Wallet } from 'ethers'
+import { Config } from '../../../common'
+import erc20Json = require('../../../contracts/build-ovm/ERC20.json')
+import upgradeableProxyJson = require('../../../contracts/build-ovm/UpgradeableProxy.json')
+
+describe('Read proxy event', () => {
+  let l2Provider: JsonRpcProvider
+  let l2Wallet: Wallet
+
+  before(async () => {
+    // l1Provider = new JsonRpcProvider(Config.L1NodeUrlWithPort())
+    // l1Signer = new Wallet(Config.DeployerPrivateKey()).connect(l1Provider))
+    const key =
+    '0xa35617f4fe630bf50024fcbe2c051d2dffe5ea19695b2d660ce4db7a5acdcc30'
+    l2Provider = new JsonRpcProvider(Config.L2NodeUrlWithPort())
+    l2Wallet = new Wallet(key).connect(l2Provider)
+    // l2Wallet = await l2Provider.getSigner()
+  })
+
+  it('should read the event correctly', async () => {
+    const initialAmount = 1000
+    const tokenName = 'OVM Test'
+    const tokenDecimals = 8
+    const TokenSymbol = 'TOK'
+
+    const implFactory = new ContractFactory(
+      erc20Json.abi,
+      erc20Json.bytecode,
+      l2Wallet
+    )
+    const impl = await implFactory.deploy(initialAmount, tokenName, tokenDecimals, TokenSymbol)
+    await impl.deployTransaction.wait()
+
+    // erc20Json.abi[0].inputs = []
+    const proxyFactory = new ContractFactory(
+      upgradeableProxyJson.abi,
+      // [...erc20Json.abi, ...upgradeableProxyJson.abi],
+      upgradeableProxyJson.bytecode,
+      l2Wallet
+    )
+    const proxy = await proxyFactory.deploy(impl.address, Buffer.from(''))
+    await proxy.deployTransaction.wait()
+
+    const contract = new Contract(proxy.address, erc20Json.abi, l2Wallet)
+
+    const _transfer = async (to, amount) => {
+      const transferTx = await contract.transfer(to, amount);
```

Comment:
@tynes it builds locally for me now but still running into `CALL_EXCEPTION` when i call transfer here 😞

Reply:
could be any of the 4 `transfer`s we call in the test, but still getting this error